### PR TITLE
Lead to customer UX enhancement

### PR DESCRIPTION
**Pull request by chdecultot**
Hi,

This is a small enhancement in the customer DocType.

Today we have the ability to set a market segment and an industry in the Lead, but these two values are not present in the Customer, where they could be very helpful for reporting purposes.

I propose to add these two fields in the Customer doctype so these values can be automatically populated when the customer is created from a lead.

I propose to add them in the section "More information":
image

Thanks in advance for your review and feedback!

### CODE REVIEW

**Comment by rmehta** [comment]
Create a new master (DocType) for "Market Segment" - Users will want to customize this for their business, so a configurable master is the best option.

**Review by rmehta — CHANGES_REQUESTED** [review]
Create new DocType "Market Segment"

**Comment by chdecultot** [comment]
Hi @rmehta,

I created a new DocType called Market Segment as requested.
Thanks a lot for your review.

Have a nice day!

File: erpnext/patches/v11_0/add_market_segments.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import frappe
+from frappe import _
+from erpnext.setup.setup_wizard.operations.install_fixtures import make_fixture_records
+
+def execute():
+	frappe.reload_doc('crm', 'doctype', 'market_segment')
+
+	frappe.local.lang = frappe.db.get_default("lang") or 'en'
+
+	records = [
```
Comment (rmehta) [inline erpnext/patches/v11_0/add_market_segments.py]: make a separate function for this and just call it?

Comment (chdecultot) [reply erpnext/patches/v11_0/add_market_segments.py]: Of course, I've juste made the split.

**Pull request merged**